packages/@office-iss/react-native-win32-tester/src/js/examples-win32/Accessibility/AccessibilityExampleWin32.tsx

```diff
@@ -223,9 +224,9 @@ const SelectionItemComponent: React.FunctionComponent<ISelectionItemComponentPro
         onAccessibilityAction={_onAccessibilityAction}
         onAccessibilityTap={_onPress}
       >
-        <Text>Item {props.value.toString()}</Text>
+        <TextWin32>{props.value.toString()}</TextWin32>
```

Comment:
If you are not doing anything win32 specific, I'd use the regular Text component.